ci-operator/step-registry/ipi/conf/vsphere/enable-template/ipi-conf-vsphere-enable-template-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+declare vsphere_datastore
+declare vsphere_url
+declare vsphere_cluster
+source "${SHARED_DIR}/vsphere_context.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+installer_bin=$(which openshift-install)
+ova_url=$("${installer_bin}" coreos print-stream-json | jq -r '.architectures.x86_64.artifacts.vmware.formats.ova.disk.location')
+echo "${ova_url}" > "${SHARED_DIR}"/ova_url.txt
+vm_template="${ova_url##*/}"
```

Comment:
Suggest to introduce a ENV var - `RHCOS_VM_TEMPLATE`, its default value is empty, so that we can allow users to override a specific image, when `RHCOS_VM_TEMPLATE` is empty, but this step is referenced in a workflow, then we can use the above logic to get a default ova vm template path.

Reply:
From my understanding, it make no difference for `template` feature in vsphere connected and disconnected segment.